content/applications/finance/fiscal_localizations/peru.rst

```diff
@@ -942,3 +942,126 @@ a scheduled action, but you can also send each invoice manually if needed.
+  for inventory reporting.
+
+   .. image:: peru/l10n-pe-type-of-existence.png
+      :alt: Select Type of Existence for a product.
+
+- **Automatic inventory valuation:** For storable goods (products for which inventory is tracked),
+  it's important to use :doc:`automatic inventory valuation
+  <../../inventory_and_mrp/inventory/product_management/inventory_valuation/inventory_valuation_config>`.
+  Once automatic inventory valuation is turned on, this valuation method can be turned on for
+  a product's :ref:`Product Category <inventory/warehouses_storage/valuation-on-product-category>`.
+
+- **Costing method:** Storable goods also need a :doc:`costing method
+  <../../inventory_and_mrp/inventory/product_management/inventory_valuation/inventory_valuation_config>`
+  that is *not* set to `Standard Price`. This is important as the Journal Entries generated from
+  stock moves will feed the PLE reports.
+
+.. _peru/reports/warehouses:
+
+Warehouses
+^^^^^^^^^^
+
+When setting up a warehouse, the :guilabel:`Annex Establishment Code` field is important for
+reporting. This code acts as a unique ID and should be different per warehouse. The number should
+only be a numeric combination, containing between 4 to 7 digits.
+
+.. image:: peru/l10n-pe-annex-establishment-code.png
+   :alt: Choose a unique ID between 4-7 numeric digits.
+
+.. _peru/reports/transfers:
+
+Inventory transfers
+^^^^^^^^^^^^^^^^^^^
+
+Transferring inventory is a key process captured in the PLE 12.1 and PLE 13.1 reports.
+:doc:`Inventory transfers <../../inventory/shipping_and_receiving/daily_operations>` include both
+incoming and outgoing shipments.
+
+When validating an inventory transfer (either on a warehouse receipt or delivery order), select the
+:guilabel:`Type of Operation` performed according to SUNAT's Table 12 for permanent inventory
+reporting.
+
+.. image:: peru/l10n-pe-type-of-operation-pe.png
+   :alt: Select a Type of Operations from SUNAT's Table 12
+
+Generate a TXT file for Permanent Inventory Kardex Reports
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+PLE 12.1 and 13.1 come as two, separate books. The books need to be downloaded in `.txt` file format
+from Odoo, and then they should be submitted to the SUNAT PLE software.
+
+On the `Inventory Valuation Report
+<https://www.odoo.com/documentation/18.0/applications/inventory_and_mrp/inventory/
+product_management/inventory_valuation/using_inventory_valuation.html#inventory-valuation-report>`,
+click the :guilabel:`PLE REPORT` button. Select the period and choose a report to export:
+either the PLE 12.1 or PLE 13.1 record. Then, Odoo will generate a `.txt` file for the chosen
+report.
+
+.. image:: peru/l10n-ple-export-button.png
+   :alt: Export Buttons selection
+
+.. note::
+   The functionality allows you to download the report in `.txt` format,
+   there is no preview or visualization like other Odoo Inventory Reports.
```

Comment:
To avoid the use of "you" and to be more direct
```suggestion
   Only a download of the report in :file:`.txt` format is available, there is no preview or
   visualization available within Odoo.
```